google/cloud/spanner_v1/instance.py

```diff
@@ -99,6 +99,15 @@ class Instance(object):
                          Cloud Console UI. (Must be between 4 and 30
                          characters.) If this value is not set in the
                          constructor, will fall back to the instance ID.
+
+    :type emulator_host: str
+    :param emulator_host: (Optional) The address for the Cloud Spanner emulator
+                          that the database of this instance should connect to.
+                          If this value is not set, the instance will connect
+                          to the Cloud Spanner endpoint.
```

Comment:
I'm curious why this is added here now.

Reply:
While adding the `labels` parameter docstring I noticed it was missing and figured the right thing would be to add it. Should this be done in a separate PR?

Reply:
Yeah probably, just to make sure that it's easy to revert the labels change if necessary.